docs/demos/delete-br1-linux-bridge.yaml

```diff
@@ -0,0 +1,6 @@
+spec:
```

Comment:
should you also add here 

```yaml
apiVersion: nmstate.io/v1alpha1
kind: NodeNetworkState
metadata:
  name: node01
```

just like in the creation?

Reply:
I broke creation adding it, we are using it with kubectl patch --type merge so they don't need that.

Fixed  create.